resources/functions/user-management/requirements.txt

```diff
@@ -0,0 +1 @@
+jsonpickle
```

Comment:
Is this necessary? I'd rather avoid packaging this up into it's own layer if we don't absolutely need it.

Reply:
this is not packaged in a layer, but with the function that needs it. I refrained from not using jsonpickle as I don't want to risk changing API contracts at this point.

Reply:
removed it